portfolio/src/main/webapp/script.js

```diff
@@ -16,7 +16,7 @@
  * Adds "Hello Naa'Oli!" to the page.
  */
 function getHelloName() {
-
-  const helloNameContainer = document.getElementById('container');
-  helloNameContainer.innerText = "Hello Naa'Oli!";
+  fetch('/data').then(response => response.json()).then((message) => {
```

Comment:
`async` and `await` are a simpler way to handle promises in modern javascript. You should do this instead:

```
async function getHelloName() {
  const response = await fetch('/data');
  document.getElementById('container').innerText = response.json().message;
}
```

https://javascript.info/async-await

Reply:
I tried to implement async and await but when I call the function in the portfolio, it gives "undefined" as the message inside the container.